website/docs: add clarification about formatting URLs to Style Guide (#13601)
goauthentik/authentik: website/docs/developer-docs/docs/style-guide.mdx

### Logical order

> - When writing procedural documentation (How Tos) the steps should follow the workflow in the UI, presenting the exact page(s) they need to navigate, and then the exact fields, tabs, etc that they need to select or fill out. Present the UI components in the document in the same order they appear in the UI.

Reviewer: What do you think?
```suggestion
- When writing procedural documentation (How-To guides), the steps should align with the UI workflow, specifying the exact pages to navigate and the precise fields, tabs, etc., to select or complete. Additionally, present UI components in the document in the same order they appear in the interface.
```

Author: It's a fine rewrite, am taking aprts of it but not all. ;-)

### Headings

Use headings (sub-titles) to break up large blocks of text, making it easier for users to navigate the content and find specific sections quickly.

### Look and feel of the docs

> In general, the visual "aesthetics" of technical documentation aim to be "lean and clean." This is achieved through concise content, such as shorter sentences and clear instructions, and a layout that maintains a clean, uncluttered appearance with limited color use and minimal large callouts or announcements. Additionally, the colors light blue and light yellow are exclusively reserved for Info and Warning callouts, respectively.

Reviewer 2: Suggestion: I don't think you need quotes around _aesthetics_. That's exactly what they are.  The motto of "lean and clean" is fine as is, but look at it without the quotes to see if it doesn't scan better, doesn't give people an out by implying some sense of irony or tounge-in-cheek.

Author: agree. no easy outs. use the darned style guide. :-)

> In general, the visual, "aesthetics" of the technical documentation is intended to be "lean and clean". Both the content (shorter sentences, concise instructions, etc) and the layout strive to have a clean, uncluttered look, with restrained use of colors and large call0outs or announcements. Relatedly, the colors used for our Info and Warning callouts, light blue and light yellow respectively, are reserved for those purposes only.

Reviewer: What do you think?
```suggestion
In general, the visual "aesthetics" of technical documentation aim to be "lean and clean." This is achieved through concise content, such as shorter sentences and clear instructions, and a layout that maintains a clean, uncluttered appearance with limited color use and minimal large callouts or announcements. Additionally, the colors light blue and light yellow are exclusively reserved for Info and Warning callouts, respectively.
```

    - Inline code snippets (e.g., `.env`).

- When handling URLs:

   - For URLs entered as values or defined in fields _italicize_ any variables within them to emphasize that placeholders require user input.

>         In markdown, use this syntax: `<kbd>https://<em>company-domain</em>/source/oauth/callback/<em>source-slug</em></kbd>`

Reviewer: Not sure on capitalization gere
```suggestion
        In Markdown, use this syntax: `<kbd>https://<em>company-domain</em>/source/oauth/callback/<em>source-slug</em></kbd>`
```

        Rendered example: <kbd>https://<em>company-domain</em>/source/oauth/callback/<em>source-slug</em></kbd>

>     - When mentioning URLs in text or within procedural instructions, omit code formatting. For instance: "In your browser, go to https://example.com."

Reviewer: If a user says "go to https://ak.company/if/flow/whatever" it will return an error as ak.company is not their real domain. We could potentially add required wording in this case? For other URLs, existing wording is good!

Reviewer: WDYM

### Examples

> When you want to show an example (say, a code snippet), start on a new line, use bold text for the word "Example", and a semi-colon, like this:

Reviewer: Mabye?
```suggestion
When you wish to provide an example (such as a code snippet), begin on a new line, use bold text for the word "Example", and a semi-colon, like this:
```
